components/scream/src/share/io/scorpio_input.cpp

```diff
@@ -279,6 +284,48 @@ void AtmosphereInput::read_variables (const int time_index)
               }}}
               break;
             }
+          case 4:
```

Comment:
This smells like something that might go in `field_utils` as well.

Reply:
Maybe, yes. The src view is not a field, but we could have a method `void deep_copy(Field<Real>& f, view_1d_type<Real> src);`. I think the `deep_copy` method of `Field` could be implemented in terms of this too...

Reply:
OTOH, implementing `Field::deep_copy(src)` in terms of the new util would create a circular dep between `field.hpp` and `field_utils.hpp`... So I might punt on this.